README.md

```diff
@@ -2,7 +2,7 @@
 
 [![Build status](https://dev.azure.com/github/Atom/_apis/build/status/Atom%20Production%20Branches?branchName=master)](https://dev.azure.com/github/Atom/_build/latest?definitionId=32&branchName=master)
 
-> Atom and all repositories under Atom will be archived on December 15, 2022. Learn more in our [official announcement](https://github.blog/2022-06-08-sunsetting-atom/)
+> Due to changes in focus The upstream Atom  and all it's repositories under the original Atom branding will be archived on December 15, 2022. Get involved here while we work on the future of Atom-community. You can learn more in their [official announcement](https://github.blog/2022-06-08-sunsetting-atom/) and get involved here.
```

Comment:
maybe change it to "Due to changes in the upstream, Atom ..."?

Reply:
I mean if it sounds better you can definitely make that change to it. Like I said this is only a starting guide for this(and I don't know how to change it now that I've made this pr tbh and still have it stay here...)

Reply:
ah.. I am not a part of the organisation and can't commit to the branch